tests/test_frame.py

```diff
@@ -655,21 +660,19 @@ def test_types_groupby_agg() -> None:
         assert_type(df.groupby("col1").agg(["min", "max"]), pd.DataFrame), pd.DataFrame
     )
     check(assert_type(df.groupby("col1").agg([min, max]), pd.DataFrame), pd.DataFrame)
+    agg_dict1: dict[Hashable, str] = {"col2": "min", "col3": "max", 0: "sum"}
```

Comment:
Does it also work when the type is `dict[str, str]`? Might need to change the keys to `HashableT` (or simply `Any`).

Reply:
Fixed

I mentioned this in my other PR, but it wasn't clear to me what the consensus type was for a column or index label. I see `Hashable` in some places, and `Scalar` in others.